### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by vita133**
made changes in plane.js and difference.js

### CODE REVIEW

File: Tasks/difference.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-    object_1[attribute_name] = object_2[attribute_name]
-    delete object_1[attribute_name]
-    }{}
-    return object_1
-}
-
-const result = diff({ a: 'uno', b: 'due' }, { a: 'uno', c: 'tre' });
+"use strict";
+const difference = (obj1, obj2) => {
+    let diffObj = { ...obj1 };
```
Comment (tshemsedinov) [inline Tasks/difference.js:5]: Prefer const

File: Tasks/difference.js [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+    for (const key in obj2) {
```
Comment (tshemsedinov) [inline Tasks/difference.js:6]: Prefer for..of

File: Tasks/difference.js [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
+      diffObj[key] = obj2[key];
+      delete diffObj[key];
+    }
+    return diffObj;
```
Comment (tshemsedinov) [inline Tasks/difference.js:10]: Don't change incoming parameters

File: Tasks/plane.js [hunk 4]
```diff
@@ -1,19 +1,17 @@
...
-    arr[i] = res[j - 1]
-    res.push(value);
-    };
-  };
-  return res
-}
+"use strict";
+const plane = (arr) => {
+  const res = [];
+  for (const value of arr) {
```
Comment (tshemsedinov) [inline Tasks/plane.js:6]: for..of over and object?

File: Tasks/difference.js (hunk 1 above)
Comment (tshemsedinov) [reply Tasks/difference.js:5]: - Use eslint and prettier
- Respect 2 space indentation

File: Tasks/plane.js (hunk 4 above)
Comment (vita133) [reply Tasks/plane.js:6]: >  what do you mean by "object"?

**Pull request closed without merging**